mlir/lib/Dialect/Rock/Transforms/GridwiseGemmToBlockwise.cpp

```diff
@@ -176,16 +176,19 @@ computeCopyPerThread(Type elementType, int64_t copyPerThread, int64_t kPerBlock,
   return std::make_pair(copyKPerThread, copyDPerThread);
 }
 
-/// Wraps the LDS buffer "buffer", which is <kOuter * d * kpack * sizeof(T) x i8
-/// into a tid x iter view, where `iter` iterates over nominal scalar indices
-/// into a buffer of type T. `buffer` will be reinterpreted as a buffer with
-/// element type vector<kpackPerThread x T> (with kpackPerThread == 1 meaning
-/// just T). The resulting view must be iterated over with a stride of no less
-/// than min(kPerThread, kpack).
+/// Wraps the LDS buffer "buffer", which is <kOuter * rotate(d) * kpack *
+/// sizeof(T) x i8 into a tid x iter view, where `iter` iterates over nominal
```

Comment:
Comment nit: I woludn't put the rotate in here, and add back the bracket after i8, to emphasize that the input is a 1D buffer.

Then I'd mention that the `d` dimension is rotated in a separate sentence